subsys/fs/Kconfig

```diff
@@ -52,6 +52,12 @@ config FUSE_FS_ACCESS
 menu "FatFs Settings"
 	visible if FAT_FILESYSTEM_ELM
 
+config FS_FATFS_EXFAT
+    bool "Enable exFAT support"
+	help
+	  Enable the exFAT format support for FatFs.
+	  After enabled the exFAT, the support of LFN is needed.
```

Comment:
`select` or `depend` keywords, which are not equivalent, allow you to set relation between Kconfig options, use them rather than notes in description.